package com.ritesh.order.exceptions;

public class EmptyCartException extends  RuntimeException{
    public EmptyCartException(String message) {
        super(message);
    }
}
